src/Compatibility/Core/src/Android/Deserializer.cs

```diff
@@ -11,37 +12,35 @@ namespace Microsoft.Maui.Controls.Compatibility.Platform.Android
 {
 	internal class Deserializer : IDeserializer
 	{
-		const string PropertyStoreFile = "PropertyStore.forms";
+		const string PropertyStoreFile = "PropertyStore.maui";
```

Comment:
Since no compatibility with `Xamarin.Forms` is expected, would it be better to use `System.Json` for serialization and deserialization? It is at least more space efficient if not faster too.

Reply:
I think that is because it is a separate dependency we have to bring in. Also, this feature is very obsolete and bringing in a dependency for a feature we basically want to delete is not worth it.

Reply:
I was under the impression that `System.Json` is part of `net6.0` and hence `net6.0-android`. Never mind then 😅. I agree, it's not worth it.